src/avram_slugify.cr

```diff
@@ -25,10 +25,8 @@ module AvramSlugify
         parameterize(candidate)
       end.reject(&.blank?)
 
-      slug_candidates.each do |candidate|
-        next if query.where(slug.name, candidate).first?
-        slug.value = candidate
-      end
+      slug_candidates.find { |candidate| query.where(slug.name, candidate).none? }
+        .tap { |candidate| slug.value = candidate }
     end
```

Comment:
In the process of remembering that I needed to make ameba ignore the usage of .any? I noticed that we were going through the whole list instead of stopping at the first one not in the database. Notice that we didn't break out of the .each after assigning the slug value.

Should be more performant for anyone that was giving a list of potential candidates to choose from.